src/std/sxml/tal/expander.ss

```diff
@@ -0,0 +1,252 @@
+(import :std/misc/alist :std/sxml/tal/toplevel :std/sxml/html/parser :std/format)
+(export #t)
+
+(def (fmt sym . args) (apply format (symbol->string sym) args))
+
+(def tal-end
+  (lambda (tag attrs parent-seed seed v?)
+    ;;(displayln "Got " tag seed " and pssed" parent-seed)
+    (if (tal-attrs? attrs)
+      (let (el (expand-tal-element tag attrs [] seed v?))
+	[(cut push-toplevel-element el) ;]
+	 (identity parent-seed) ...])
+      (html-end tag attrs parent-seed seed v?))))
+
+(def (html->tal-form html) (sxml->tal-form (html->sxml html end: tal-end)))
+    
+(def (tal-attr? attr attrs)
+  (let (alist (member attr attrs (lambda (a b) (eq? a (and (pair? b) (car b))))))
+    (and alist (car alist))))
+
+(def (expand-tal-element tag attrs parent-seed seed v?)
+  (def els [[tal:define? . expand-tal:define]
```

Comment:
hash table?

Reply:
That's actually a good idea for external reasons in case one wants to add another tag. I like alists and plists way too much it seems.